packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -683,6 +693,7 @@ export class ContainerRuntime
 			runtimeOptions = {},
 			containerScope = {},
 			containerRuntimeCtor = ContainerRuntime,
+			initializeEntryPoint = async (containerRuntime) => containerRuntime,
```

Comment:
this looks wrong. it shouldn't return containerRuntime

Reply:
the default should probably just be undefined

Reply:
Ah, meant to leave that as a comment, yeah, if we wanted a default or not... I wasn't fully convinced myself so happy to remove it.